controller/tenant.go

```diff
@@ -175,22 +185,33 @@ func (c *TenantController) Update(ctx *app.UpdateTenantContext) error {
 	return ctx.Accepted()
 }
 
-func (c *TenantController) loadUserTenantConfiguration(token *jwt.Token, config openshift.Config) (openshift.Config, error) {
-	authHeader := token.Raw
-	if len(c.usersURL) > 0 {
-		url := strings.TrimSuffix(c.usersURL, "/") + "/user"
-		status, body, err := openshift.GetJSON(config, url, authHeader)
+// loadUserTenantConfiguration loads the tenant configuration for `auth`,
+// allowing for config overrides based on the content of his profile (in auth) if the user is allowed
+func (c *TenantController) loadUserTenantConfiguration(ctx context.Context, config openshift.Config) (openshift.Config, error) {
+	token := goajwt.ContextJWT(ctx)
+	// use the toggle service to verify that the user is allowed to change is profile based on the settings passed in the URL
+	if len(c.authURL) > 0 && c.toggleClient.IsEnabled(token, toggles.UserUpdateTenantFeature, false) {
```

Comment:
This is not really a good use of Toggles. Toggles != Permanent User config.

https://github.com/fabric8-services/fabric8-tenant/issues/229

Reply:
Sorry, I'm not following you here: the goal of the change is to allow (or not) user config override, based on the toggle state for the user. The user's tenants will be initialised with the default versions for Che/Jenkins/etc, unless toggle is enabled and user has some specific settings in his/her profile.

Reply:
@xcoulon Yes, I get that. But this is not a temporary setting, right? It's not related to 'let's try something out' or 'while we deploy this new feature' type of setting. It's a permanent feature of the user that he has more rights or whatever. Since it's permanent it's the wrong use of the Toggle Service.

Reply:
@aslakknutsen ok, got it, so using the toggle service is not the right way to handle this permission. So, should we set a flag in the user's profile or use `auth` to check the permission to override the profile ?

Reply:
@xcoulon I would add some type of 'group' to the "tenant" table. That way we can use the info when doing automatic updates as well. E.g. roll out to group N first, then wait 24h and unless stopped, roll out to group X.. etc etc

Reply:
@aslakknutsen not following your idea: we need to restrict permission to override the default config to a list of users who would need to test PRs. How do you expect to handle this in the `tenant` table and how do you expect to manage the data (I'm thinking about an admin UI, here)

Reply:
@aslakknutsen what about using an attribute in KC records to handle a whitelist of users who are allowed to override the config ?

Reply:
@xcoulon The UserProfile service or what ever would mark a user to belong to some group (some service account is allowed to change that data on behalf of a user). 

The problem with using KC for this is that it would require a user token to look it up... which causes issues for auto update and extra pressure on KC.

Reply:
what auto-update ? I thought this would be managed by an admin ?
if KC is not a good candidate, what about rules handled by our `auth` service, then ?

Reply:
@xcoulon Auto update of tenants.

Reply:
@alexeykazakov @aslakknutsen ok, so what solution should I use in the mean time (i.e., until auth supports groups/team permissions), then ?

Reply:
We could enable custom jenkins/che only for specific users (by UUID) for now.

Reply:
yes, that's the point of this issue and using the toggle service was one way to implement it (i.e, verify that the user's ID belongs to a whitelist). However, I understand @aslakknutsen's argument that toggle service should be used to enable/disable features that are not released yet, which is not the use-case for this solution. Hence my question: where do we keep track of the permission ?